### ISSUE TITLE
B009 inconsistent behaviour with default argument

### ISSUE THREAD

**Issue by vchaptsev** [PyCQA/flake8-bugbear#120]
I guess it should be consistent

Example:
```python
# no warnings
getattr(obj, 'attr', None)

# B009
getattr(obj, 'attr', default=None)
```

Additional context:
python3.7.3
flake8-bugbear==20.1.4

**Comment by vchaptsev** [comment]
Sorry, idk why I've tried to do that, `getattr() takes no keyword arguments`
